### ISSUE TITLE
HTTPS/SSL Support

### ISSUE THREAD

**Issue by FireDrunk** [jlambert121/jlambert121-kibana#72]
Is it possible to change the kibana.yml.erb template to include options for SSL?

from the docs:
```
# SSL for outgoing requests from the Kibana Server (PEM formatted)
server.ssl.key: /path/to/your/server.key
server.ssl.cert: /path/to/your/server.crt
```

Would be much appreciated!

**Comment by jlambert121** [comment]
@FireDrunk how about taking a stab at providing those parameters?  PRs are greatly appreciated!

**Comment by smetj** [comment]
@FireDrunk https://github.com/jlambert121/jlambert121-kibana/pull/73

**Comment by jlambert121** [comment]
Closed with #75

**Comment by FireDrunk** [comment]
Thanks guys, much appreciated!
